From 58ebf98fb7abdc126169c48a5606c448ecfc8270 Mon Sep 17 00:00:00 2001
From: Anas Nashif <anas.nashif@intel.com>
Date: Sat, 2 Feb 2013 17:25:21 -0800
Subject: [PATCH 08/20] remove manifest macro

---
 packaging/pam.spec | 1 -
 1 file changed, 1 deletion(-)

diff --git a/packaging/pam.spec b/packaging/pam.spec
index a6ddc34..0d8aebe 100644
--- a/packaging/pam.spec
+++ b/packaging/pam.spec
@@ -149,7 +149,6 @@ fi
 %{_pamconfdir}/other
 
 %files -n pam-modules-extra
-%manifest pam.manifest
 %{_moduledir}/pam_access.so
 %{_moduledir}/pam_debug.so
 %{_moduledir}/pam_echo.so
-- 
1.8.1.4

