From 3b802af3505daec915c7c740b83a9d388a64015d Mon Sep 17 00:00:00 2001
From: Anas Nashif <anas.nashif@intel.com>
Date: Sat, 2 Feb 2013 17:33:53 -0800
Subject: [PATCH 10/20] remove manifest

---
 packaging/pam.spec | 1 -
 1 file changed, 1 deletion(-)

diff --git a/packaging/pam.spec b/packaging/pam.spec
index 0d8aebe..6ef0aad 100644
--- a/packaging/pam.spec
+++ b/packaging/pam.spec
@@ -177,7 +177,6 @@ fi
 %{_moduledir}/pam_warn.so
 
 %files devel
-%manifest pam.manifest
 %{_includedir}/security/*
 %{_libdir}/libpam.so
 %{_libdir}/libpam_misc.so
-- 
1.8.1.4

