From 0bf5af91e99d30942ea89172f0aa3dfda19691f7 Mon Sep 17 00:00:00 2001
From: Anas Nashif <anas.nashif@intel.com>
Date: Sat, 2 Feb 2013 17:34:13 -0800
Subject: [PATCH 11/20] remove non-existant manifest macro

---
 packaging/pam.changes | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packaging/pam.changes b/packaging/pam.changes
index d23bfa3..eedb6cb 100644
--- a/packaging/pam.changes
+++ b/packaging/pam.changes
@@ -1,3 +1,6 @@
+* Sat Feb 02 2013 Anas Nashif <anas.nashif@intel.com> submit/trunk/20130203.012535@3b802af
+- remove non-existant manifest macro
+
 * Sat Feb 02 2013 Anas Nashif <anas.nashif@intel.com> accepted/trunk/20130117.013515@58ebf98
 - remove manifest macro
 
-- 
1.8.1.4

