From: Quanxian Wang <quanxian.wang@intel.com>
Date: Fri, 25 Oct 2013 16:23:04 +0800
Subject: Fix ivi building issue

Change-Id: Ide344ca8061311e403a0878a78743e666153106b
Signed-off-by: Quanxian Wang <quanxian.wang@intel.com>
---
 configure.ac | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configure.ac b/configure.ac
index 98ef1d7..99b4288 100644
--- a/configure.ac
+++ b/configure.ac
@@ -612,6 +612,10 @@ EFL_SELECT_WINDOWS_VERSION
 
 ### Checks for libraries
 
+if test "${want_evas_engine_wayland_egl}" = "yes"; then
+   CFLAGS="$CFLAGS -I/usr/include/freetype2 -Ifribidi"
+fi
+
 EVIL_DLFCN_LIBS=""
 if test "${have_wince}" = "xyes"; then
    EFL_ADD_LIBS([EVIL], [-lws2])
