diff --git a/display-manager-run.service b/display-manager-run.service
index 2242f04..dff3897 100644
--- a/display-manager-run.service
+++ b/display-manager-run.service
@@ -10,7 +10,7 @@ EnvironmentFile=/etc/sysconfig/weston
 
 #adding the capability to configure ttys
 #may be needed if the user 'display' doesn't own the tty
-#CapabilityBoundingSet=CAP_SYS_TTY_CONFIG 
+CapabilityBoundingSet=CAP_SYS_TTY_CONFIG CAP_SYS_ADMIN
 
 [Install]
 WantedBy=graphical.target
