diff --git a/smart/backends/rpm/pm.py b/smart/backends/rpm/pm.py
index ecd9f4e..f5c3051 100644
--- a/smart/backends/rpm/pm.py
+++ b/smart/backends/rpm/pm.py
@@ -216,6 +216,7 @@ class RPMPackageManager(PackageManager):
         if sysconf.get("rpm-order"):
             ts.order()
         probfilter = rpm.RPMPROB_FILTER_OLDPACKAGE
+        probfilter |= rpm.RPMPROB_FILTER_IGNOREARCH
         if force or reinstall:
             probfilter |= rpm.RPMPROB_FILTER_REPLACEPKG
             probfilter |= rpm.RPMPROB_FILTER_REPLACEOLDFILES
@@ -303,7 +304,7 @@ class RPMCallback:
             self.fd = os.open(path, os.O_RDONLY)
             setCloseOnExec(self.fd)
             return self.fd
-        
+
         elif what == rpm.RPMCALLBACK_INST_CLOSE_FILE:
             if self.fd is not None:
                 os.close(self.fd)
