diff --git a/pkgmgr_plugin/CMakeLists.txt b/pkgmgr_plugin/CMakeLists.txt
index b8a0488..00f8428 100644
--- a/pkgmgr_plugin/CMakeLists.txt
+++ b/pkgmgr_plugin/CMakeLists.txt
@@ -12,11 +12,7 @@ FOREACH(flag ${privileges_CFLAGS})
 ENDFOREACH(flag)
 
 INCLUDE_DIRECTORIES(
-    /usr/include
-    /usr/include/glib-2.0 
-    /usr/include/libxml2
-    /usr/include/package_manager
-    /usr/include/dlog
+    "${pkg_INCLUDE_DIRS}"
     "${CMAKE_SOURCE_DIR}/client/inc/"
     "${CMAKE_SOURCE_DIR}/common/inc/"
     )

 
