From aa90c99fd90e1944684c7fe268611136d568e691 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?K=C3=A9vin=20THIERRY?= <kevin.thierry@open.eurogiciel.org>
Date: Mon, 18 Aug 2014 14:39:54 +0200
Subject: [PATCH] meta-tizen: Yocto: correct build issue
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Kévin THIERRY <kevin.thierry@open.eurogiciel.org>
---
 data/Makefile.am | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/data/Makefile.am b/data/Makefile.am
index 1e80530..7a7afc1 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -56,9 +56,11 @@ $(tablefiles): interpolation2.text
 
 bigram.db: $(textual_model_data)
 	$(RM) $(binary_model_data)
-	../utils/storage/gen_binary_files --table-dir $(top_srcdir)/data
-	../utils/storage/import_interpolation --table-dir $(top_srcdir)/data < $(top_srcdir)/data/interpolation2.text
-	../utils/training/gen_unigram --table-dir $(top_srcdir)/data
+	cd $(top_srcdir)/utils/storage
+	gen_binary_files --table-dir $(top_srcdir)/data
+	import_interpolation --table-dir $(top_srcdir)/data < $(top_srcdir)/data/interpolation2.text
+	cd $(top_srcdir)/utils/training
+	gen_unigram --table-dir $(top_srcdir)/data
 
 phrase_index.bin pinyin_index.bin $(binfiles): bigram.db
 
-- 
1.8.1.4

