diff --git a/CMakeLists.txt b/CMakeLists.txt
index 22e73a7..28102f2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,15 +19,17 @@ MESSAGE("Build type: ${CMAKE_BUILD_TYPE}")
 
 # Set required packages
 INCLUDE(FindPkgConfig)
-SET(AUL-1_PKG_CHECK_MODULES dlog ecore libprivilege-control app-checker rua glib-2.0 ecore-input evas vconf pkgmgr-info privacy-manager-client)
+SET(AUL-1_PKG_CHECK_MODULES dlog ecore libprivilege-control app-checker rua glib-2.0 vconf pkgmgr-info privacy-manager-client)
 OPTION(WITH_WAYLAND "Build with wayland" OFF)
-IF(WITH_WAYLAND)
-    ADD_DEFINITIONS("-DWAYLAND")
-    pkg_check_modules(pkgs REQUIRED ${AUL-1_PKG_CHECK_MODULES} ecore-wayland)
-ELSE(WITH_WAYLAND)
-    ADD_DEFINITIONS("-DX11")
-    pkg_check_modules(pkgs REQUIRED  ${AUL-1_PKG_CHECK_MODULES} x11 ecore-x)
-ENDIF(WITH_WAYLAND)
+pkg_check_modules(pkgs REQUIRED ${AUL-1_PKG_CHECK_MODULES} )
+
+#IF(WITH_WAYLAND)
+#    ADD_DEFINITIONS("-DWAYLAND")
+#    pkg_check_modules(pkgs REQUIRED ${AUL-1_PKG_CHECK_MODULES} ecore-wayland)
+#ELSE(WITH_WAYLAND)
+#    ADD_DEFINITIONS("-DX11")
+#    pkg_check_modules(pkgs REQUIRED  ${AUL-1_PKG_CHECK_MODULES} x11 ecore-x)
+#ENDIF(WITH_WAYLAND)
 
 pkg_check_modules(libpkgs REQUIRED dlog bundle dbus-glib-1 ail xdgmime app-checker libtzplatform-config)
 PKG_CHECK_MODULES(PKGS REQUIRED glib-2.0 gio-2.0 dlog bundle)
@@ -56,7 +58,7 @@ SET(CMAKE_C_FLAGS_RELEASE "-O2")
 
 SET(CMAKE_SKIP_BUILD_RPATH true)
 
-# Get uname value to set 'TARGET' definition 
+# Get uname value to set 'TARGET' definition
 # TODO: Is this needed?
 FIND_PROGRAM(UNAME NAMES uname)
 EXEC_PROGRAM("${UNAME}" ARGS "-m" OUTPUT_VARIABLE "ARCH")
@@ -121,11 +123,9 @@ add_executable(amd
 		am_daemon/amd_status.c
                 )
 SET(AUL-1_LINK_LIBRARY amd aul_mods app-checker-server rua glib-2.0 bundle ail aul)
-IF (WITH_WAYLAND)
+
 target_link_libraries(${AUL-1_LINK_LIBRARY} ${pkgs_LDFLAGS})
-ELSE(WITH_WAYLAND)
-target_link_libraries(${AUL-1_LINK_LIBRARY} utilX ${pkgs_LDFLAGS})
-ENDIF(WITH_WAYLAND)
+
 
 SET(REL_AGENT daemon-manager-release-agent)
 ADD_EXECUTABLE(${REL_AGENT} agent/${REL_AGENT}.c)
