diff --git a/test/app_test.c b/test/app_test.c
index d92da2a..ce75ddc 100755
--- a/test/app_test.c
+++ b/test/app_test.c
@@ -22,11 +22,12 @@
 #include <poll.h>
 #include <stdio.h>
 #include <unistd.h>
+#include <sys/time.h>
 #include "aul.h"

 /* ecore-glib integration */

-#include <Ecore.h>
+//#include <Ecore.h>

 extern int aul_listen_app_dead_signal(int (*func) (int, void *), void *data);

@@ -44,7 +45,7 @@ void do_resume()
 	printf("=================================\n");
 }

-static Eina_Bool send_result(void *data)
+static int send_result(void *data)
 {
 	bundle *kb;
 	bundle *res_b;
@@ -94,7 +95,7 @@ int do_start(void *data)

 	bundle_iterate(kb, prt_bundle, NULL);

-	ecore_timer_add(5, send_result, bundle_dup(kb));
+	//ecore_timer_add(5, send_result, bundle_dup(kb));

 	return 0;
 }
@@ -134,7 +135,7 @@ int app_launch_handler(int pid, void *data)
 __attribute__ ((visibility("default")))
 int main(int argc, char **argv)
 {
-	ecore_init();
+	//ecore_init();

 	do_create();

@@ -152,7 +153,7 @@ int main(int argc, char **argv)
 		exit(0);
 	}

-	ecore_main_loop_begin();
+	//ecore_main_loop_begin();
 	return 0;
 }

diff --git a/test/aul_test.c b/test/aul_test.c
index c912951..ef40ab7 100755
--- a/test/aul_test.c
+++ b/test/aul_test.c
@@ -24,7 +24,7 @@
 #include <stdio.h>
 #include <sys/time.h>
 #include <unistd.h>
-#include <Ecore.h>
+//#include <Ecore.h>

 #include "menu_db_util.h"
 #include "aul.h"
@@ -121,9 +121,9 @@ static void cb_func(bundle *kb, int is_cancel, void *data)
 		bundle_iterate(kb, prt_recvd_bundle, NULL);
 	}

-	if ((strcmp(cmd, "launch_res") == 0)
+	/*if ((strcmp(cmd, "launch_res") == 0)
 	    || (strcmp(cmd, "open_svc_res") == 0))
-		ecore_main_loop_quit();
+		ecore_main_loop_quit();*/
 }

 int open_test()
@@ -175,14 +175,14 @@ static test_func_t scn_func[] = {
 	{"n", launch_test, "launch_test", ""}
 };

-static Eina_Bool run_all_test(void *data)
+static int run_all_test(void *data)
 {
 	static int pos = 0;
 	int ret;

 	if (pos > sizeof(scn_func) / sizeof(test_func_t) - 1) {
 		printf("all internal test done\n");
-		ecore_main_loop_quit();
+		//ecore_main_loop_quit();
 		return 0;
 	}

@@ -204,7 +204,7 @@ static Eina_Bool run_all_test(void *data)

 int all_test()
 {
-	ecore_timer_add(2, run_all_test, NULL);
+	//ecore_timer_add(2, run_all_test, NULL);
 	return 0;
 }

@@ -661,7 +661,7 @@ void print_usage(char *progname)
 		"cmd is internal purpose\n");
 }

-static Eina_Bool run_func(void *data)
+static int run_func(void *data)
 {
 	callfunc(cmd);

@@ -669,8 +669,8 @@ static Eina_Bool run_func(void *data)
 	    || strcmp(cmd, "dbuslaunch") == 0
 	    || strcmp(cmd, "open_svc_res") == 0)
 		return 0;
-	else
-		ecore_main_loop_quit();
+	/*else
+		ecore_main_loop_quit();*/

 	return 0;
 }
@@ -682,7 +682,7 @@ int main(int argc, char **argv)
 		exit(0);
 	}

-	ecore_init();
+	//ecore_init();

 	cmd = argv[1];
 	gargc = argc;
@@ -694,9 +694,9 @@ int main(int argc, char **argv)
 	/*aul_listen_app_dead_signal(dead_tracker,NULL); */
 	/*aul_listen_app_dead_signal(NULL,NULL); */

-	ecore_idler_add(run_func, NULL);
+	/*ecore_idler_add(run_func, NULL);

-	ecore_main_loop_begin();
+	ecore_main_loop_begin();*/

 	return 0;
 }
diff --git a/test/launch_app.c b/test/launch_app.c
index 47b9b2d..6cd1cfe 100755
--- a/test/launch_app.c
+++ b/test/launch_app.c
@@ -24,7 +24,7 @@
 #include <unistd.h>
 #include <sys/types.h>

-#include <Ecore.h>
+//#include <Ecore.h>
 #include "aul.h"

 #define ROOT_UID 0
@@ -110,7 +110,7 @@ static int __launch_app_dead_handler(int pid, void *data)
 	return 0;
 }

-static Eina_Bool run_func(void *data)
+static int run_func(void *data)
 {
 	int pid = -1;
 	char *str = NULL;
@@ -121,12 +121,12 @@ static Eina_Bool run_func(void *data)

 		if( str && strcmp(str, "SYNC") == 0 ) {
 			aul_listen_app_dead_signal(__launch_app_dead_handler, pid);
-		} else {
+		} /*else {
 			ecore_main_loop_quit();
-        }
+        }*/
 	} else {
 		printf("... launch failed\n");
-		ecore_main_loop_quit();
+		//ecore_main_loop_quit();
 	}

 	if (kb) {
@@ -145,16 +145,16 @@ int main(int argc, char **argv)
 		exit(EXIT_FAILURE);
 	}

-	ecore_init();
+	//ecore_init();

 	gargc = argc;
 	gargv = argv;

 	aul_launch_init(NULL, NULL);

-	ecore_idler_add(run_func, NULL);
+	//ecore_idler_add(run_func, NULL);

-	ecore_main_loop_begin();
+	//ecore_main_loop_begin();

 	return 0;
 }
diff --git a/test/open_app.c b/test/open_app.c
index d9b677c..49eb4f5 100755
--- a/test/open_app.c
+++ b/test/open_app.c
@@ -24,7 +24,7 @@
 #include <unistd.h>
 #include <sys/types.h>

-#include <Ecore.h>
+//#include <Ecore.h>
 #include "aul.h"

 #define ROOT_UID 0
@@ -104,13 +104,13 @@ static int __launch_app_dead_handler(int pid, void *data)
 {
 	int listen_pid = (int) data;

-	if(listen_pid == pid)
-		ecore_main_loop_quit();
+	/*if(listen_pid == pid)
+		ecore_main_loop_quit();*/

 	return 0;
 }

-static Eina_Bool run_func(void *data)
+static int run_func(void *data)
 {
 	int pid = -1;
 	char *str = NULL;
@@ -124,9 +124,9 @@ static Eina_Bool run_func(void *data)

 	if( str && strcmp(str, "SYNC") == 0 ) {
 		aul_listen_app_dead_signal(__launch_app_dead_handler, pid);
-	} else {
+	} /*else {
 		ecore_main_loop_quit();
-	}
+	}*/

 	if (kb) {
 		bundle_free(kb);
@@ -143,16 +143,16 @@ int main(int argc, char **argv)
 		exit(EXIT_FAILURE);
 	}

-	ecore_init();
+	//ecore_init();

 	gargc = argc;
 	gargv = argv;

 	aul_launch_init(NULL, NULL);

-	ecore_idler_add(run_func, NULL);
+	//ecore_idler_add(run_func, NULL);

-	ecore_main_loop_begin();
+	//ecore_main_loop_begin();

 	return 0;
 }
--
1.7.9.5

