diff -ruN gst-plugins-bad-1.4.5-orig/configure.ac gst-plugins-bad-1.4.5/configure.ac
--- gst-plugins-bad-1.4.5-orig/configure.ac	2015-11-13 16:54:06.573553133 +0200
+++ gst-plugins-bad-1.4.5/configure.ac	2015-11-13 16:58:17.085548765 +0200
@@ -1971,7 +1971,7 @@
 dnl **** Wayland ****
 translit(dnm, m, l) AM_CONDITIONAL(USE_WAYLAND, true)
 AG_GST_CHECK_FEATURE(WAYLAND, [wayland sink], wayland , [
-  PKG_CHECK_MODULES(WAYLAND, wayland-client >= 1.4.0 wayland-scanner, [
+  PKG_CHECK_MODULES(WAYLAND, wayland-client >= 1.4.0, [
     HAVE_WAYLAND="yes" ], [ HAVE_WAYLAND="no"
   ])
 ])
