Index: Mesa-10.3.4/configure.ac
===================================================================
--- Mesa-10.3.4.orig/configure.ac
+++ Mesa-10.3.4/configure.ac
@@ -832,7 +832,7 @@ AM_CONDITIONAL(HAVE_SHARED_GLAPI, test "
 # Build the pipe-drivers as separate libraries/modules.
 # Do not touch this unless you know what you are doing.
 # XXX: Expose via configure option ?
-enable_shared_pipe_drivers=no
+enable_shared_pipe_drivers=yes
 
 dnl
 dnl Driver specific build directories
